src/mutation.ts

```diff
@@ -292,7 +336,7 @@ export class Mutation {
    * @returns {object}
    * @private
    */
-  static parse(mutation) {
+  static parse(mutation: Mutation): JsonObj {
```

Comment:
Should this be [`IMutateRowRequest`](https://github.com/googleapis/nodejs-bigtable/blob/master/proto/bigtable.d.ts#L709) ?